src/routes.ts

```diff
@@ -1,43 +1,85 @@
+export type SubRoute = {
+  name: string
+  path: string
+  hash?: string
+}
+
 export type Route = {
   name: string
   path: string
   hash?: string
+  title: string
```

Comment:
What is the purpose of the `title` property? 🧐

You already have the `name` property that is sufficient to describe the route.